module1/homework/grayscaleImages/compression.hpp

```diff
@@ -0,0 +1,15 @@
+#pragma once
+#include <array>
+#include <vector>
+#include <cctype>
```

Comment:
I think that newline between pragma and standard library includes would be nice.